plugins/modules/source_control/gitlab/gitlab_project.py

```diff
@@ -114,6 +114,50 @@
       - Used to create a personal project under a user's name.
     type: str
     version_added: "3.3.0"
+  allow_merge_on_skipped_pipeline:
+    description:
+      - allow merge when skipped pipelines exist
```

Comment:
```suggestion
      - Allow merge when skipped pipelines exist.
```
Descriptions must always be proper sentences. Please also adjust the other ones.